spec/features/user_views_book_page_spec.rb

```diff
@@ -25,4 +25,45 @@
     expect(page).to have_content('June 19, 2015')
     expect(page).to have_css("img[src='http://example.com/cool_cover.jpg']")
   end
+
+  describe "didn't sign in" do
+    it "can't leave a review for a book" do
+      book = create :book
+      visit book_path(book)
+      expect(page).to have_content('Please sign in to leave a review')
+      expect(page).to_not have_content('Leave a review')
+    end
+  end
+
+  describe 'signed in' do
+    it "sees other users reviews" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.